### PR TITLE
Feature/ecom timer

### PR DESCRIPTION
**Pull request by kody-joao**
Adding ecom protocol to set a countdown timer and an expiration time

### CODE REVIEW

File: src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto [hunk 1]
```diff
@@ -29,6 +29,12 @@ message PaymentInitiationRequest {
   optional string payer_ip_address = 10; // The payer IP address used for risk checks, also required for 3D Secure 2 transactions.
   optional string payer_locale = 11; // The language code and country code to specify the language to display the payment pages. It will default to en_GB if not set.
   optional bool tokenise_card = 12; // defaults false
+  optional ExpirySettings expiry = 13; // Nested message for expiry settings
+
+  message ExpirySettings {
+    bool show_timer = 1; // Display a countdown timer to the user in the payment page, default is false
```
Comment (afabris-kodypay) [inline src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto:35]: Is not this redundant if the message itself is optional? We could say that ExpirySettings.expiring_seconds is set only when expiry is set and given with optional the server can understand if the property is set or not we don't need the boolean? Unless I'm missing a use case where it's important to set it to false?

Comment (kody-joao) [reply src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto:35]: This is about the UI to display or not the timer, which is unrelated with the real expiration time.
The ecom might want to set an expiration of 1 hour to pay, and not display the countdown.

File: docs/CODEOWNERS [hunk 2]
```diff
@@ -3,4 +3,4 @@
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
 
-* @sishbi @sarah-jay-peper @cargillchantry @geodavies @afabris-kodypay @rfontes17
+* @sishbi @sarah-jay-peper @cargillchantry @geodavies @afabris-kodypay @rfontes17 @kody-joao
```
Comment (rfontes17) [inline docs/CODEOWNERS:6]: Could you add Ioannis?

File: src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto (hunk 1 above)
Comment (kody-joao) [reply src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto:35]: Feel free to suggest an alternative way where the user can set an expiration of 3600 seconds and not display the timer on the screen, or simulating our current PBL behaviour of 70 days expirations and not showing timer.

Comment (afabris-kodypay) [reply src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto:35]: Now makes sense. Thanks :)

**Review by afabris-kodypay — APPROVED** [review]

**Pull request merged**